angular/src/lib/select/select.component.ts

```diff
@@ -170,7 +170,7 @@ export class SelectComponent implements AfterContentChecked, ControlValueAccesso
   @Input() optionLabel: string;
   /** @prop show the filter search | false */
   @Input() filter: boolean = false;
-  /** @prop set which key the filter will use | 'label' */
+  /** @prop set which key or keys the filter will use, filtering by multiple keys example: 'name,id' | 'label' */
```

Comment:
Let's make this a type `string | string[]` so that you can have an array of filters instead of a comma separated string.
Then you pass the array on to the `FilterUtils.filter()` method instead of converting the string into an array.
This will also make the @Input type more clear for the api.

Reply:
Thanks!
Updated so either works, string or array of strings 
[filterBy]="['name','delete']"
filterBy='delete'